.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: ruby
+rvm:
+  - 2.2
+
+sudo: false
+
+install:
+  - bundle install --path vendor/bundle
```

Comment:
add `bundle config build.nokogiri --use-system-libraries` to avoid building nokogiri during build.